### PR TITLE
Update README with updated websockets URL.

### PR DESCRIPTION
**Pull request by cckim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,15 +15,16 @@ That's it! `foo` can be a `@rest` query, it could be `@dbquery`, it could be a f
...
 6. Setup .env; `echo 'APIKEY='$(stepzen whoami --apikey) > .env`  (`stepzen whoami --apikey` retrieves your admin key, it shoud look something like: `graphql::local.net...`)
 7. `node client.js` and see the magic. You will see the price of `ETHBTC` change and reflected back on your console.
 
 ### Steps for the StepZen cloud service
 1. Download this repo and `cd subscription-blog/graphql`. 
 2. Make sure you've registered for an account and setup your cli.  [Getting Started](https://stepzen.com/getting-started) has instructions on how to do this.
-3. `cd graphql && stepzen deploy` deploys the `graphql` schema in `index.graphql` to StepZen (on your local machine) and the endpoint is `http://localhost:9000/api/subscription/__graphql`
+3. `cd graphql && stepzen deploy` deploys the `graphql` schema in `index.graphql` to StepZen (on your local machine) and the websockets endpoint is `wss://ACCOUNT.stepzen.net/stepzen-subscriptions/api/subscription/__graphql`.  Note: the client code below will use the correct wss endpoint if you set STEPZEN_ACCOUNT per step 6.
+http://localhost:9000/api/subscription/__graphql`
```
Comment (ddebrunner) [inline README.md]: misplaced?

**Review by ddebrunner — CHANGES_REQUESTED** [review]
misplaced url I think

File: README.md (hunk 1 above)
Comment (cckim) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -15,15 +15,15 @@ That's it! `foo` can be a `@rest` query, it could be `@dbquery`, it could be a f
...
 6. Setup .env; `echo 'APIKEY='$(stepzen whoami --apikey) > .env`  (`stepzen whoami --apikey` retrieves your admin key, it shoud look something like: `graphql::local.net...`)
 7. `node client.js` and see the magic. You will see the price of `ETHBTC` change and reflected back on your console.
 
 ### Steps for the StepZen cloud service
 1. Download this repo and `cd subscription-blog/graphql`. 
 2. Make sure you've registered for an account and setup your cli.  [Getting Started](https://stepzen.com/getting-started) has instructions on how to do this.
-3. `cd graphql && stepzen deploy` deploys the `graphql` schema in `index.graphql` to StepZen (on your local machine) and the endpoint is `http://localhost:9000/api/subscription/__graphql`
+3. `cd graphql && stepzen deploy` deploys the `graphql` schema in `index.graphql` to StepZen (on your local machine) and the websockets endpoint is `wss://ACCOUNT.stepzen.net/stepzen-subscriptions/api/subscription/__graphql`.  Note: the client code below will use the correct wss endpoint if you set STEPZEN_ACCOUNT per step 6.
```
Comment (cckim) [inline README.md]:
```suggestion
3. `cd graphql && stepzen deploy` deploys the `graphql` schema in `index.graphql` to the StepZen managed service and the websockets endpoint is `wss://ACCOUNT.stepzen.net/stepzen-subscriptions/api/subscription/__graphql`.  Note: the client code below will use the correct wss endpoint if you set STEPZEN_ACCOUNT per step 6.
```

**Review by ddebrunner — APPROVED** [review]

**Pull request merged**